website/content/en/status/report-2023-07-2023-09/bsd-cafe.adoc

```diff
@@ -1,38 +1,41 @@
-=== Introducing the BSD.cafe
+=== Introducing the BSD Cafe project
 
 Links: +
-link:https://wiki.bsd.cafe[BSD Cafe Project Homepage] URL: link:https://wiki.bsd.cafe[] +
-link:https://mastodon.bsd.cafe[BSD Cafe Mastodon Instance] URL: link:https://mastodon.bsd.cafe[]
+link:https://wiki.bsd.cafe/[BSD Cafe project homepage] URL: link:https://wiki.bsd.cafe/[] +
+link:https://mastodon.bsd.cafe/[BSD Cafe Mastodon instance] URL: link:https://mastodon.bsd.cafe/[]
 
 Contact: Stefano Marinelli <stefano@dragas.it>
 
 We are thrilled to unveil the inaugural component of the BSD Cafe project!
-Months ago, when I first registered the link:https://bsd.cafe[bsd.cafe] domain, I envisioned a themed bar where friends, acquaintances, and patrons could gather for casual conversations about *BSD systems, Linux, and Open-Source technology.
+
+Months ago, when I first registered the `bsd.cafe` domain, I envisioned a themed bar where friends, acquaintances, and patrons could gather for casual conversations about ++*++BSD systems, Linux, and open-source technology.
```

Comment:
Please restore the link macro here: I prefer to give more information rather than less. Moreover, the `.cafe` tld is not very common, so having something clickable helps people less familiar with uncommon domains understand what `bsd.cafe` actually is.

Reply:
Resolved by [`ea87cb7` (#255)](https://github.com/freebsd/freebsd-doc/pull/255/commits/ea87cb78f8a2fc4fbbad6dd6e636b249eb798505).